### PR TITLE
:construction_worker: Simplify passing coverage/tox envvars

### PR DESCRIPTION
**Pull request by sergei-maertens**
Since we use the codecov action to upload coverage reports, it already automatically merges the coverage data from the entire matrix. We do not need to manage envvars manually.

### CODE REVIEW

File: dotgithub/workflows/ci.yml [hunk 1]
```diff
@@ -58,6 +55,8 @@ jobs:
         run: |
           pip install build --upgrade
           python -m build
+
+      # TODO: switch to verified publishers
       - name: Publish a Python distribution to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
```
Comment (Viicos) [inline dotgithub/workflows/ci.yml:61]: Do you not trust the `PyPA` enough? :yum:

Comment (sergei-maertens) [reply dotgithub/workflows/ci.yml:61]: No I do, but there is a new mechanism where we don't even have to provide a PyPI token anymore, instead PyPI is configured to trust github actions as a verified publisher: https://docs.pypi.org/trusted-publishers/using-a-publisher/

Comment (Viicos) [reply dotgithub/workflows/ci.yml:61]: Ah nice wasn't aware of this feature

**Review by Viicos — APPROVED** [review]
I haven't added the coverage `exclude_also` config in the previous PR, so maybe you'd like to add it here

**Comment by sergei-maertens** [comment]
> I haven't added the coverage `exclude_also` config in the previous PR, so maybe you'd like to add it here

You can create a separate PR, I needed to get my immediate needs fixed so I can get started on a new library :wink:

**Pull request merged**